samples/openshift-cloud-terminal.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: workspace.che.eclipse.org/v1alpha1
+kind: Workspace
+metadata:
+  name: openshift-terminal
+  annotations:
+    org.eclipse.che.workspace/immutable: "true"
+  labels:
+    console.openshift.io/cloudshell: "true"
```

Comment:
just a marker for OpenShift Console to pick up the workspace as a terminal

Reply:
Isn't this label used by the console to store username? Does putting "true" here work?

Reply:
OpenShift Console uses `console.openshift.io/cloudshell-user` annotation to store username and `console.openshift.io/cloudshell` is supposed to have true/false value. 
It eventually makes sense to rename it to `console.openshift.io/openshift-terminal` I think, but it's example that shows the current state.

P.S. I had a conversation with Abhi about username annotation OpenShift Console uses and we agreed that it makes sense to just reuse `org.eclipse.che.workspace/creator` annotation with UID controller provisions.

Reply:
If you're renaming "cloud-shell" to "cloud-terminal" then I suspect you need to rename here too.